### PR TITLE
Add SR2019 london kickstart

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: kickstart-london.tex [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+\newcommand{\assessorname}{Andrew Barrett-Sprot}
+\newcommand{\assessoremail}{abarrett-sprot@studentrobotics.org}
+\newcommand{\assessmentdate}{October 20, 2018}
+
+
+\newcommand{\activityname}{Student Robotics 2019 - Kickstart}
+\newcommand{\activitydate}{November 10, 2018}
+\newcommand{\activitytime}{All day}
+\newcommand{\activitylocation}{
+    Office for Thread, 1 Alie St, London E1 8DE, UK
```
Comment (PeterJCLaw) [inline kickstart-london.tex]: Perhaps "Thread's Offices"? "Office for Thread" sounds odd to me. If this is meant to be an address, then maybe drop the "Offices" part entirely?

File: kickstart-london.tex [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    autonomous robots. The robots must perform the simple task of locating and
+    moving marked boxes around an arena.
+
+    The students are aged 16--18 and will be supervised by at least one teacher
+    from their school at all times.
+
+    The programme is run by the charity Student Robotics, who
+    will be considered event staff at this and future events.
+
+    The schedule of the event is:
```
Comment (PeterJCLaw) [inline kickstart-london.tex]: I'm not sure how the schedule relates to this being a risk-assessment?
If we are keeping it, maybe it should reflect the proposed schedule -- that the talks are about 45 minutes and then the teams are into prep/μgames.

Comment (PeterJCLaw) [reply kickstart-london.tex]: Or perhaps merely summarise the overview of the day -- that it contains some talks and then a long workshop session?
I don't know how much detail is expected for this sort of thing though, so if the detail is needed then obviously lets just update it to be correct.

**Review by PeterJCLaw — APPROVED** [review]
Seems good to me.
Do we want to mention the presence of a sink and a **hot** water tap (it's one of those ones where you push a button and it emits ~boiling water for making hot drinks with, rather than an ordinary sink tap) in the London venue?

File: kickstart-london.tex (hunk 2 above)
Comment (Adimote) [reply kickstart-london.tex]: I suspect having a schedule is overkill, we should really just highlight the fact that there's a workshop session

**Pull request merged**